django/forms/models.py

```diff
@@ -52,6 +52,10 @@ def construct_instance(form, instance, fields=None, exclude=None):
             continue
         if exclude and f.name in exclude:
             continue
+        if not f.null and cleaned_data[f.name] is None:
+            # It's not the job of construct_instance to raise a possible ValueError
+            # here, let's delegate that to the validation step.
```

Comment:
If I understand correctly "let's delegate..." -> "that will be done in instance.full_clean()"?

I'd like to check a few more things later today so you can hold off on that update until I finish review.